addons/web_unsplash/controllers/main.py

```diff
@@ -81,12 +82,18 @@ def slugify(s):
         for key, value in unsplashurls.items():
             url = value.get('url')
             try:
-                if not url.startswith('https://images.unsplash.com/'):
+                if not url.startswith('https://images.unsplash.com/') and not request.env.registry.in_test_mode():
                     logger.exception("ERROR: Unknown Unsplash URL!: " + url)
                     raise Exception(_("ERROR: Unknown Unsplash URL!"))
-                req = requests.get(url)
-                if req.status_code != requests.codes.ok:
-                    continue
+
+                # FIXME check why `requests.get("http://127.0.0.1:8069/")`
+                # timeout/never finish in test mode
+                if request.env.registry.in_test_mode():
+                    req = Mock(content=b'\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00\x01\x00\x00\x00\x01\x01\x03\x00\x00\x00%\xdbV\xca\x00\x00\x00\x03PLTE\x00\x00\x00\xa7z=\xda\x00\x00\x00\x01tRNS\x00@\xe6\xd8f\x00\x00\x00\nIDAT\x08\xd7c`\x00\x00\x00\x02\x00\x01\xe2!\xbc3\x00\x00\x00\x00IEND\xaeB`\x82')
+                else:
+                    req = requests.get(url)
+                    if req.status_code != requests.codes.ok:
+                        continue
```

Comment:
@qsm-odoo @JKE-be (anyone reviewing this)
This diff shouldn't be here but..
Somehow the `requests.get("http://127.0.0.1:8069/")` when performed in the test never finish, the build timeout at 61min.
But this is an important fix as this restore unsplash which is not working at all right now..
So would be cool if can merge it with this and fix it afterward.. we would have the unsplash working again and being tested (so it shouldnt break that easily again)

Note that the tour works fine when run from the browser.. I guess the test environment is altering requests.get in some ways.. I will figure

Reply:
solution / less ugly hack found

Reply:
Fixed, thanks to JKE
Use static img to mock instead of controller one as local image instead of unsplash -> contourne l'erreur